pkgs/development/python-modules/git-remote-codecommit/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k, botocore }:
+
+buildPythonPackage rec {
+  pname = "git-remote-codecommit";
+  version = "1.15.1";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "052j1a4a9gccyq6wrzih81jgxx9ayqqdh42iy72zibgrji5d96fd";
+  };
+
+  propagatedBuildInputs = [ botocore ];
```

Comment:
There are no tests included. Please try to checkout from source and check if they have unit tests, and try to run them. Unit tests give a good indication that they package has a high degree of validity and correctness given the python package set.

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.

```

----------------------------------------------------------------------
Ran 0 tests in 0.000s
```

Reply:
The source contains 23 tests. Switching to the GitHub repo fixed it.